server/NewBank.java

```diff
@@ -62,12 +75,12 @@ public synchronized String processRequest(CustomerID customer, String request, N
 		return "FAIL";
 	}
 
-	public HashMap<String, Customer> getCustomers() {
+	public HashMap<String, newbank.server.Customer> getCustomers() {
 		return customers;
 	}
 
-	public String getID(Customer c) {
-		for (Entry<String, Customer> entry : customers.entrySet()) {
+	public String getID(newbank.server.Customer c) {
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it